manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CloudFoundry.CLI.v8
+PackageVersion: 8.7.2
+PackageLocale: en-US
+Publisher: Cloud Foundry Foundation
+PublisherUrl: https://cli.cloudfoundry.org/
+PublisherSupportUrl: https://github.com/cloudfoundry/cli/issues
+# PrivacyUrl: 
+# Author: 
+PackageName: Cloud Foundry CLI
```

```suggestion
PackageName: Cloud Foundry CLI v8
```
We can offer a better package name now since we control what is written to Control Panel in case of portable installers.